modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -224,6 +226,12 @@
     /** System view description for page lists. */
     public static final String CACHE_GRP_PAGE_LIST_VIEW_DESC = "Cache group page lists";
 
+    /** System view name for partition states. */
+    public static final String PART_STATES_VIEW = "partitionStates";
+
+    /** System view description for partition states. */
+    public static final String PART_STATES_VIEW_DESC = "Partition states";
```

Comment:
Can we write a more detailed description:
"Distribution of the cache group partitions across cluster nodes"